docs/cli.rst

```diff
@@ -0,0 +1,45 @@
+Command Line Interface
+======================
+
+Nameko ships with a command line interface to make hosting and interacting with services as easy as possible.
+
+Running a Service
+-----------------
+
+::
+
+    $ nameko run [<module>[:<ServiceClass>]]
+
+Discover and run a service class.
+
+
+Interacting with a running service
+----------------------------------
+
+::
+
+    $ nameko shell
+
+Launch an interactive python shell for working with remote nameko services. This is a regular interactive interpreter, with a special module ``n`` added
+to the builtin namespace, providing the ability to make RPC calls and dispatch events.
+
+Making an RPC call to "target_service":
+
+::
```

Comment:
don't know if pygments need a hand with guessing python-console here. if we're keeping the banner, maybe `emphasize-lines` (http://sphinx-doc.org/markup/code.html) could help